README.md

```diff
@@ -33,7 +37,22 @@ The mock method takes three arguments:
 
 The mock method executes insert, then update, and finally findOne and returns the result of findOne.
 
-Each argument is optional and we have some smart behavior to handle non-existant arguments
+Each argument is optional and we have some smart behavior to handle non-existant arguments:
 - `insert`, if not specified we'll grab the first part of the `update` operator
 - `update`, if not specified we'll skip the update, if a single argument is specified, we'll use the id returned by our insert operation as the first argument (the query)
-- `find`, if not specified we'll use the id returned by our insert operation
\ No newline at end of file
+- `find`, if not specified we'll use the id returned by our insert operation
+
+**The `mockMulti` method takes four arguments:**
+
+- `insert` an array of arguments to be passed to the mockCollection's insert method
+- `update` an array of arguments to be passed to the mockCollection's update method
+- `flags` an object to be passed to the mockCollection's update method
+- `find` an array of arguments to be passed to the mockCollection's find method
+
+The mockMulti method executes insert, then update, and finally find and returns the result of find().fetch().
+
+Each argument is optional:
+- `insert`, if not specified we'll grab the first part of the `update` operator
+- `update`, if not specified we'll skip the update, if a single argument is specified, we'll use the ids returned by our insert operation as the first argument (the query)
```

Comment:
Let's not support getting the query from the insert argument.

The `mockMulti` differs from the `mock` method in that it will more accurately represent what happens with the real operation. Most operations will be simpler to mock using the `mock` method because it works on only one document, and the developer most likely only cares about what happens to that single document, however some operations may affect multiple documents, in which case we need the `mockMulti` method.

For example, imagine the user wishes to simulate the results of this operation:

`mycollection.update({kind: 'new'}, {fee: 20}, {multi: true})`

The `mock` operator is not able to fully simulate the result of this operation because it only inserts and returns a single document. The developer should instead use this code to simulate the result of this operation: `mycollection.mockMulti([{kind: 'new'}, {fee: 20}, {multi: true}])` which will find all `{kind: 'new'}` documents, insert them into the mock collection, update them, and return the result.

However the `mockMulti` method is potentially very expensive, for example, consider this operation: `mycollection.mockMulti([{}, {$set: {initialized: true}}, {multi: true}])`, which will load the entire collection into memory!!

Let's fix the `mockMulti` method as described above, and add documentation to this effect:

1. You should generally use the `mock` method, especially for allow/deny rules and other cases where you know that only one document will be affected (or only care about one document).
2. You should use the `mockMulti` method to test the behavior of update operations which may affect multiple documents
3. You should use the `mockMulti` method with care on the server, it will load all affected documents into memory! (This is not an issue on the client where all documents reside in memory anyway)